CHANGES.md

```diff
@@ -100,6 +100,8 @@
 * Support for X source added (Java/Python) ([BEAM-X](https://issues.apache.org/jira/browse/BEAM-X)).
 * Add streaming support to SnowflakeIO in Java SDK ([BEAM-9896](https://issues.apache.org/jira/browse/BEAM-9896  ))
 * Support reading and writing to Google Healthcare DICOM APIs in Python SDK ([BEAM-10601](https://issues.apache.org/jira/browse/BEAM-10601))
+* Add dispositions for SnowflakeIO.write ([BEAM-10343](https://issues.apache.org/jira/browse/BEAM-10343))
```

Comment:
@youngoli Could I also ask you to change these lines (103, 104, 140, 141 as well? I've noticed that snowflake dispositions and cross-language read was added in 2.24, not 2.23 (it was pushed at 2.23 but was merged after the 2.23 release)
It would also be great to move snowflake.py from apache_beam.io.external to apache_beam.io but I don't know how much changes would you like to do. But we would avoid backward compatibility issues with this change.

Reply:
Hey Piotr. Option 1 is easily doable, but option 2 and 3 are not so much. Even though the changes are small and unlikely to break anything, adding them now would require building an entirely new RC. But moreso, the Beam release guide has some pretty strict expectations for a cherry-pick and I don't think this fits (https://beam.apache.org/contribute/release-guide/#review-cherry-picks). In particular, this doesn't seem to be a regression from a previous release, and it seems to be an update to a new feature, which can wait until 2.25. If there's an urgent reason to get it in ASAP that I'm missing, then let me know, but otherwise I think I'll limit the release changes to just fixing the changelog.

Reply:
No problem at all - as I said option 1) is fully sufficient. Thanks! :)

Reply:
I see them in https://github.com/apache/beam/pull/12745/commits/f47574a56276f582c85f3455a140fd18c92cdb3e

Thanks a lot again!